src/Analytics.php

```diff
@@ -452,16 +460,24 @@ class Analytics
      * @param bool $isSsl
      * @throws \InvalidArgumentException
      */
-    public function __construct($isSsl = false)
+    public function __construct($isSsl = false, $isDisabled = false)
     {
         if (!is_bool($isSsl)) {
             throw new \InvalidArgumentException('First constructor argument "isSSL" must be boolean');
         }
 
+        if (!is_bool($isDisabled)) {
+            throw new \InvalidArgumentException('Secondt constructor argument "isDisabled" must be boolean');
+        }
+
         if ($isSsl) {
             $this->uriScheme .= 's';
             $this->endpoint = str_replace('www', 'ssl', $this->endpoint);
         }
+        
+        if ($isDisabled) {
```

Comment:
Since you already validated that the value is a boolean, you should remove the conditional and just assign it to isDisabled.

`$this->isDisabled = $isDisabled;`